packages/cli/test/composites.test.ts

```diff
@@ -62,7 +62,7 @@ describe('composites', () => {
       ])
       const output = create.stdout.toString()
       expect(output.includes('"version":"1.0"')).toBe(true)
-      expect(output.includes('"indices":{"Post"')).toBe(true)
+      expect(output.includes('"indices":{"')).toBe(true)
```

Comment:
Post is present to verify that we are not generating empty indices.

Reply:
The check still works with the presence of `"`, the test was failing because `Profile` was the first model returned, the test shouldn't assume any specific ordering here